cmd/clusterctl/examples/openstack/provider-component/user-data/ubuntu/templates/master-user-data.sh

```diff
@@ -8,10 +8,7 @@ NAMESPACE={{ .Machine.ObjectMeta.Namespace }}
 MACHINE=$NAMESPACE
 MACHINE+="/"
 MACHINE+={{ .Machine.ObjectMeta.Name }}
-CONTROL_PLANE_VERSION={{ .Machine.Spec.Versions.ControlPlane }}
 CLUSTER_DNS_DOMAIN={{ .Cluster.Spec.ClusterNetwork.ServiceDomain }}
-POD_CIDR={{ .PodCIDR }}
-SERVICE_CIDR={{ .ServiceCIDR }}
```

Comment:
you deleted this , but line 62 below is 
CLUSTER_DNS_SERVER=$(prips ${SERVICE_CIDR} | head -n 11 | tail -n 1)
and this is some issue

Reply:
Same as above regarding ` .Cluster.Spec.ClusterNetwork.ServiceDomain` It's now handled via kubeadm